### PR TITLE
HTML Task

### PR DESCRIPTION
**Pull request by Fiyin985**

### CODE REVIEW

**Comment by Fiyin985** [comment]
Ive done the PRD

File: submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE HTML>
+<html>
+<title>
+Product Page 	
+</title>
+<head>
+	
+	<p> <img src="tte.jpg" height="200" width="2500"> <img style="float:right;" src="OIP.jpg" height="100"width="300"></p>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html:8]: These elements should not exist in the head element.

File: submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	
+</head>
+  
+<body>
+	<meta charset="utf-8">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html:13]: This meta element should be in the head element.

File: submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE HTML>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/aboutMe/PRD for E-Commerce Using HTML.html:1]: Styling is not part of the criteria for this project. I think you should remove all styling added to this file.

**Comment by Fiyin985** [comment]
What should i change the pull request name to?

**Comment by ToluAkin** [comment]
> What should i change the pull request name to?

[6] Develop a Product page using HTML

**Pull request closed without merging**